gnucash/import-export/import-main-matcher.cpp

```diff
@@ -1909,6 +1909,20 @@ get_peer_acct_names (Split *split)
     return retval;
 }
 
+static bool
+acct_recndate_warning (time64 trans_date, const Account* acc)
+{
+    time64 recn_date;
+    if (xaccAccountGetReconcileLastDate (acc, &recn_date))
+        return trans_date > recn_date;
+
+    return false;
+}
+
+const char* recn_date_warning = N_("The import date is earlier than reconcile date. \
```

Comment:
`static const char*` and inside `refresh_model_row` so that it isn't global and lives in the data segment instead of having to be created on the stack at every invocation.